src/Http/Response.php

```diff
@@ -624,7 +624,7 @@ protected function _setContent()
         );
 
         if (in_array($this->_status, [304, 204])) {
-            $this->body('');
+            $this = $this->withStringBody('');
```

Comment:
How would one fix this one?

Reply:
With this, the body() call might actually be fine, no?